tools/src/main/resources/ui/NOTICE

```diff
@@ -0,0 +1,68 @@
+RAPIDS plugin for Apache Spark
+Copyright (c) 2022, NVIDIA CORPORATION
+
+This product includes software that depends on 3rd library available under MIT License
```

Comment:
Should it be `3rd party library` or `3rd party libraries`  instead of `3rd library` ?